app/api/v2/managers/operation_api_manager.py

```diff
@@ -20,3 +49,53 @@ async def get_operation_object(self, operation_id: str, access: dict):
         if operation.match(access):
             return operation
         raise JsonHttpForbidden(f'Cannot view operation due to insufficient permissions: {operation_id}')
+
+    async def setup_operation(self, data: dict, access: BaseWorld.Access):
+        """Applies default settings to an operation if data is missing."""
+        planner_id = data.pop('planner', {}).get('id', '')
+        data['planner'] = await self._construct_and_dump_planner(planner_id)
+        adversary_id = data.pop('adversary', {}).get('adversary_id', '')
+        data['adversary'] = await self._construct_and_dump_adversary(adversary_id)
+        fact_source_id = data.pop('source', {}).get('id', '')
+        data['source'] = await self._construct_and_dump_source(fact_source_id)
+        operation = OperationSchema().load(data)
+        # Add Agents to Operation, set access, and update start time.
```

Comment:
let's get rid of this comment and change the `update_operation` function name in `c_operation.py` to `update_operation_agents`. I believe the only other location it's called is in contact_svc.py.